test/utils/math/Math.test.js

```diff
@@ -15,6 +30,132 @@ contract('Math', function () {
     this.math = await Math.new();
   });
 
+  describe('with flag', function () {
```

Comment:
I'd remove this `with flag` block. 

Every other test is at the same level and this `with flag` block indicates there's a group of functions, which made sense for the difference between `add` and `tryAdd` in the old SafeMath.

I don't think that's necessary here.